src/index.ts

```diff
@@ -1,64 +1,7 @@
-import GlobeScene from './components/scene';
-import GeoJSONType from './enums/geojson';
-import Marker from './components/marker';
-import Bar from './components/bar';
+import GlobeScene from "./components/globe";
```

Comment:
```suggestion
import GlobeScene from "./components/scene";
```

The scene needs to be imported from scene, not globe.